exercises/square-root/description.md

```diff
@@ -0,0 +1,9 @@
+Given a natural radicand, return its square root.
+
+Check out the Wikipedia pages on [square root](https://en.wikipedia.org/wiki/Square_root) and [methods of computing square roots](https://en.wikipedia.org/wiki/Methods_of_computing_square_roots).
+
+Note that the term "radicand" refers to the number for which the root is to be determined. That is, it is the number under the root symbol.
```

Comment:
Maybe it would make sense to have this explanation be given just after the first line in which radicand was introduced?

Reply:
Done